### ISSUE TITLE
Unable to install Podman on Ubutu 20.04

### ISSUE THREAD

**Issue by kyleadawson** [containers/podman#16760]
/kind bug

**Description**
Unable to install podman on Ubutu 20.04

**Steps to reproduce the issue:**

1. Follow steps on podman site - https://podman.io/getting-started/installation
2. This was working this week and for the past 1 year, just started to break last day or so.  

**Describe the results you received:**
curl -fsSL https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/xUbuntu_$(lsb_release -rs)/Release.key \
>   | gpg --dearmor \
>   | sudo tee /etc/apt/keyrings/devel_kubic_libcontainers_unstable.gpg > /dev/null
curl: (22) The requested URL returned error: 404 

**Describe the results you expected:**
Podman gets installed

```

**Have you tested with the latest version of Podman and have you checked [the Podman Troubleshooting Guide](https://github.com/containers/podman/blob/main/troubleshooting.md)?**


Yes

**Comment by wapson** [comment]
I believe this is also the issue for another Ubuntu releases (I've faced that in i.e 18.04). It would be good to fix this relatively quickly due to fact that this problem may affect a lot of environments.

**Comment by yogugl** [comment]
Similiarly on Debian 11 https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_11/Release.key
I get an expired key
pub   rsa2048 2018-08-03 [SC] [**expired: 2022-12-06**]
      2472D6D0D2F66AF87ABA8DA34D64390375060AA4
uid                      devel:kubic OBS Project <devel:kubic@build.opensuse.org>

**Comment by veita** [comment]
Could it be that `unstable` isn't supported by Ubuntu 20.04 and 18.04?

**Comment by yogugl** [comment]
Please try this link:
https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04/Release.key 
It looks like the URLs have changed.

**Comment by veita** [comment]
To me it seems that `unstable` corresponds to Podman 4 while `stable` is Podman 3.

https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/amd64/
https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04/amd64/

If Podman 4 would not be supported by Ubuntu 20.04 for some reason (e.g. Kernel version requirements) you could not use the `unstable` branch with Ubuntu 20.04 and you probably had to change `unstable` to `stable` in your URL.

**Comment by kyleadawson** [comment]
Yes I have tried switching the url to unstable.  The issue is the certificate that comes back has expired


We started with the stable that then redirects to the unstable and then that goes to a URL That's a 404.


Playing with all the features we get to a state where I don't get a 404 within the cert that comes back is expired. Looks like it expired on December 5th.



________________________________
From: Alexander Veit ***@***.***>
Sent: Sunday, December 11, 2022 5:07:13 PM
To: containers/podman ***@***.***>
Cc: Kyle Dawson ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [containers/podman] Unable to install Podman on Ubutu 20.04 (Issue #16760)

To me it seems that unstable corresponds to Podman 4 while stable is Podman 3. https: //download. opensuse. org/repositories/devel: /kubic: /libcontainers: /stable/xUbuntu_20. 04/amd64/ https: //download. opensuse. org/repositories/devel: /kubic: /libcontainers: /unstable/xUbuntu_22. 04/amd64/
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.

ZjQcmQRYFpfptBannerEnd

To me it seems that unstable corresponds to Podman 4 while stable is Podman 3.

https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/amd64/<https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/amd64/>
https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04/amd64/<https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04/amd64/>

If Podman 4 would not be supported by Ubuntu 20.04 for some reason (e.g. Kernel version requirements) you could not use the unstable branch with Ubuntu 20.04 and you probably had to change unstable to stable in your URL.

**Comment by kyleadawson** [comment]
Per the doc it seems to be correct, I even change from stable to unstable.  The core issue is that the cert has expired.




Kyle Dawson
Principal Automation Technical Specialist
IBM Technology
US National Market -  North America Top
Phone: 1-312-725-0299

From: Alexander Veit ***@***.***>
Sent: Sunday, December 11, 2022 4:26 PM
To: containers/podman ***@***.***>
Cc: Kyle Dawson ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [containers/podman] Unable to install Podman on Ubutu 20.04 (Issue #16760)

Could it be that unstable isn't supported by Ubuntu 20. 04 and 18. 04? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread. Message ID: <containers/podman/issues/16760/1345660565@ github. com>
ZjQcmQRYFpfptBannerStart
This Message Is From an External Sender
This message came from outside your organization.
ZjQcmQRYFpfptBannerEnd

Could it be that unstable isn't supported by Ubuntu 20.04 and 18.04?
